docker-compose.yml

```diff
@@ -110,7 +112,7 @@ services:
   ####                                        KEYCLOAK                                       ###
   ##############################################################################################
   keycloak:
-    image: quay.io/keycloak/keycloak
+    image: quay.io/keycloak/keycloak:legacy
```

Comment:
`docker-compose up` did not work locally for keycloak or redis unless I made these 2 small changes.